.github/workflows/kb_sdk-actions.yml

```diff
@@ -55,16 +56,29 @@ jobs:
         run: |
           kb-sdk init -l $LANGUAGE_TOKEN -u user --example SampleApp
 
-      - name: test ${{ matrix.test_type }} repo
+      - name: test ${{ matrix.test_type }} repo, using test.cfg for auth
+        if: matrix.auth == 'token'
+        run: |
+          cd SampleApp
+          kb-sdk test || true
+
+      - name: test ${{ matrix.test_type }} repo, using test.cfg for auth
+        if: matrix.auth == 'token'
         env:
```

Comment:
Does this env line need to be here given it's a token in file test?

Reply:
Yes -- otherwise the `sed` command will not be able to add the token to the test.cfg.